Plugins/GeoModel/src/GeoModelModuleSplitter.cpp

```diff
@@ -0,0 +1,86 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Plugins/GeoModel/GeoModelModuleSplitter.hpp"
+
+#include "Acts/Surfaces/AnnulusBounds.hpp"
+#include "Acts/Surfaces/DiscSurface.hpp"
+#include "Acts/Utilities/Logger.hpp"
+
+#include <iostream>
+
+namespace Acts {
+
+auto GeoModelModuleSplitter::split(DetectorElementPtr detElement,
```

Comment:
There is a trailing return type

Reply:
And you need that because of the nested typedef? Ok. Consider either removing the typedef as per my other comment or explicitly typing out the full type name on the left side of the signature.